queries/ruby/locals.scm

```diff
@@ -20,17 +20,26 @@
 ; OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 ; SOFTWARE.
 
-;;; DESCLARATIONS AND SCOPES
+;;; DECLARATIONS AND SCOPES
 ((method) @scope
  (set! scope-inherits false))
+(class) @scope
 
 [
  (block)
  (do_block) 
  ] @scope
 
-(method_parameters (identifier) @definition.function)
-(lambda_parameters (identifier) @definition.function)
+(identifier) @reference
+(constant) @reference
+(instance_variable) @reference
+
+(module name: (constant) @definition.namespace)
+(class name: (constant) @definition.namespace)
```

Comment:
@definition.type ?

Reply:
Is that what we are using for classes? If so than yes, in Ruby they have little difference to modules.

Reply:
All other languages use @defintion.type for classes

Reply:
Noted, I've updated the query.